src/pretix/base/models/items.py

```diff
@@ -799,6 +801,13 @@ class ItemVariation(models.Model):
         help_text=_('If set, this will be displayed next to the current price to show that the current price is a '
                     'discounted one. This is just a cosmetic setting and will not actually impact pricing.')
     )
+    require_approval = models.BooleanField(
+        verbose_name=_('Buying this product requires approval'),
+        default=False,
+        help_text=_('If this product is part of an order, the order will be put into an "approval" state and '
+                    'will need to be confirmed by you before it can be paid and completed. You can use this e.g. for '
+                    'discounted tickets that are only available to specific groups.'),
+    )
```

Comment:
The new field should also be exposed in the REST API, which requires you to change the serializer (ItemVariationSerializer) as well as the test cases and docs for the applicable API endpoints

Reply:
Regarding „product“ in help-text, see above.

Reply:
I think I found a nice solution, but please check again :)